ci/pipelines/osspi-wavefront-sdk-go/set_pipeline.sh

```diff
@@ -3,13 +3,14 @@ set -efuo pipefail
 
 which fly || (
   echo "This requires fly to be installed"
-  echo "Download the binary from https://github.com/concourse/concourse/releases or from the Runway Concourse: https://runway-ci.eng.vmware.com"
+  echo "Download the binary from https://github.com/concourse/concourse/releases or from the Runway Concourse: https://runway-ci-sfo.eng.vmware.com"
   exit 1
 )
 
+TARGET=runway-sfo
 fly -t runway sync || (
   echo "This requires the runway target to be set"
-  echo "Create this target by running 'fly -t runway login -c https://runway-ci.eng.vmware.com -n tobs-k8s-group'"
+  echo "Create this target by running 'fly -t "${TARGET}" login -c https://runway-ci-sfo.eng.vmware.com -n cpo-team-troll'"
```

Comment:
change request: how about we create a new runway group just for the Wavefront SDKs?